### PR TITLE
Refactor/about page

### PR DESCRIPTION
**Pull request by stephanie-roe**
## What was changed:
- Refined styling to about page to implement parallax effect according to wireframe. 

## Explain the reason for changes:
(If there is no spec file how were things tested)
- Changes implemented to allow for a more seamless UX. 
- Updates tested and all Cypress tests passing.

### CODE REVIEW

**Comment by Abumpus1** [comment]
Nice job setting up parallax!

File: src/Components/AboutPage.js [hunk 1]
```diff
@@ -17,14 +20,11 @@ const AboutPage = () => {
...
-        <Gap></Gap>
         <JoinCTAContainer className="join-cta-container">
-          <CtaImage
-            className="cta-image"
-            src={require("../assets/bannerImages/ctaImage1.png")}
-            alt="brightly-colored woven rounds"
-          />
+        
+          <Lottie className="cta-animation" style={{height: "300px", margin: "40px"}} animationData={require("../assets/animations/clay-crafting.json")} loop={true}/>
           <CtaContainer>
```
Comment (Wperricone) [inline src/Components/AboutPage.js:28]: Does the Cta have to do with Parallax?

**Review by Wperricone — COMMENTED** [review]
Great work! Looks so good!

File: src/Components/AboutPage.js [hunk 2]
```diff
@@ -123,25 +130,32 @@ const CtaP = styled.p`
...
-  align-items: center;
+const AboutContainer = styled.div`
+  height: 100vh;
+  overflow-y: scroll;
+  overflow-x: hidden;
+  perspective: 10px;
+  margin-bottom: 20px;
 `;
+
+const ParallaxImage = styled.img`
```
Comment (JoeRupp) [inline src/Components/AboutPage.js:141]: Super cool! Great work getting that figured out. I'm going to have to look into this more.

**Pull request merged**